spec/rails_helper.rb

```diff
@@ -5,6 +5,22 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+
+OmniAuth.config.test_mode = true
+  OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new({
+      :provider => "google_oauth2",
+      :uid => "123456789",
+      :id => '1',
```

Comment:
maybe remove the :id line and use the :uid line instead (looks like that's what we'll be getting back)

Reply:
This was a change we made so that we can get users and artwork together that we actually have in our mocks and stubs. Will change once integrated with backend.